src/irmin-pack/unix/io.ml

```diff
@@ -168,7 +169,6 @@ module Unix = struct
            usage is safe. *)
         let buf = Bytes.unsafe_of_string s in
         let () = Util.really_write t.fd off buf 0 len in
-        Index.Stats.add_write len;
```

Comment:
I just remembered that the stats here are used as a quick and dirty way to aggregate IO activity for the benchmark summary.

We need to double check the "official" summary printer but you can see how it is used for the `bench/irmin-pack` tree replay. https://github.com/mirage/irmin/blob/1f046ddaedf3532bb236f7d19510182c5948b9d6/bench/irmin-pack/trace_stat_summary_pp.ml#L384-L390

I am definitely in favor of cleaning this up and making it more clear that these are not "index" stats. Perhaps your new Io stats can help the summary aggregate the index activity as well.